hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -887,15 +886,26 @@
       tests: []
     }
     {
-      name: chip_sw_pwrmgr_power_glitch_reset
+      name: chip_sw_pwrmgr_aon_power_glitch_reset
+      desc: '''Verify the cold boot sequence through the wiggling of `por_rst_n`.
+
+            The vcaon_supp_i AST input is pulsed low to trigger corresponding PORs.
+            This ensures that both FSMs are properly reset on the POR signal. The check is that
+            the processor ends up running. Also verify, the rstmgr records POR in `reset_info` CSR.
+            '''
```

Comment:
Probably worth clarifying POR_RST_NI pin / AON power supply glitch are randomly picked for a number of times.